app/controllers/worldpay_missed_payment_forms_controller.rb

```diff
@@ -35,6 +31,16 @@ def authorize_user
     authorize! :record_worldpay_missed_payment, @resource
   end
 
+  def redirect_after_successful_submit
```

Comment:
I wanted to make some comment about how it's a shame we essentially have this code in the concern, so why not use the concern. But then I tried and couldn't make it work well.

So this is more a pointer to me to discuss this bit face-to-face.